test/test-network/conf/25-ip6tnl-external.netdev

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
Maybe delete the file in this commit since it gets updated in the next one? At this point in the commit stack it doesn't have `External=true` yet

Reply:
Ouch.

Reply:
The file is moved to the last commit.